chain/src/main/scala/org/bitcoins/chain/blockchain/ChainHandler.scala

```diff
@@ -486,6 +453,101 @@ case class ChainHandler(
     }
     loop(Future.successful(to), Vector.empty)
   }
+
+  def isMissingChainWork: Future[Boolean] = {
+    for {
+      first100 <- blockHeaderDAO.getBetweenHeights(0, 100)
+      first100MissingWork = first100.nonEmpty && first100.exists(
+        _.chainWork == BigInt(0))
```

Comment:
won't this be problematic if someone starts to calculate chain work but doesn't finish?